docs/sources/write/markdown-guide/index.md

```diff
@@ -147,7 +147,7 @@ When rendered, the preceding table displays as follows:
 | ------------- | ------------- |
 | Cell one data | Cell two data |
 
-## Numbered lists
+## Numbered
```

Comment:
Numbered is an adjective in this context, so I would not remove "lists" here.

Reply:
Incorporated feedback, thank you.